File: CS/MyGridCtrl/MyGridHandler.cs
using DevExpress.XtraGrid.Views.Grid;
using DevExpress.XtraGrid.Views.BandedGrid;
using DevExpress.XtraGrid.Views.BandedGrid.Handler;

namespace DevExpress.S170736
{
    public class MyGridHandler : BandedGridHandler {
        public MyGridHandler(BandedGridView gridView)
            : base(gridView) {
        }
        protected override XtraGrid.Menu.GridViewFooterMenu CreateGridViewFooterMenu(GridView gridView) {
            return new MyGridViewFooterMenu(gridView);
        }
    }
}

File: VB/MyGridCtrl/MyGridHandler.vb
Imports DevExpress.XtraGrid.Views.Grid
Imports DevExpress.XtraGrid.Views.BandedGrid
Imports DevExpress.XtraGrid.Views.BandedGrid.Handler

Namespace DevExpress.S170736
    Public Class MyGridHandler
        Inherits BandedGridHandler
        Public Sub New(ByVal gridView As BandedGridView)
            MyBase.New(gridView)
        End Sub
        Protected Overrides Function CreateGridViewFooterMenu(ByVal gridView As GridView) As XtraGrid.Menu.GridViewFooterMenu
            Return New MyGridViewFooterMenu(gridView)
        End Function
    End Class
End Namespace
